code-of-conduct-languages/hi.md

```diff
@@ -0,0 +1,27 @@
+# योगदानकर्ता आचार संहिता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, तथा एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो मुद्दों को रिपोर्ट करने, फीचर अनुरोध पोस्ट करने, दस्तावेज अपडेट करने, पुल अनुरोध या पैच सबमिट करने एवं अन्य गतिविधियों के माध्यम से योगदान करते हैं।
```

Comment:
`खुले और स्वागत करने वाले समुदाय` can be replaced with `कम्युनिटी`

We don't have localise it word by word, it should express what it actually means as it will be understand in Hindi language.

Reply:
So I need to correct that right?

Reply:
@anubha-v-ardhan what do you think?

Reply:
I think, we should keep the word `समुदाय`. However `खुले और` can be removed as it sounds little abnormal.
So, it will be `स्वागत करने वाले समुदाय`

Reply:
Done

Reply:
खुले और स्वागत करने वाले समुदाय is replaced with कम्युनिटी

Reply:
As @anubha-v-ardhan mentioned, this around sounds good `स्वागत करने वाले समुदाय` or you can go with `कम्युनिटी`.

Reply:
Done